arch/amd64.sh

```diff
@@ -1,5 +1,5 @@
 #!/bin/bash
 ##arch/amd64.sh: Build definitions for amd64.
 ##@copyright GPL-2.0+
-CFLAGS_COMMON_ARCH=('-fomit-frame-pointer' '-march=x86-64' '-mtune=sandybridge' '-msse2')
+CFLAGS_COMMON_ARCH=('-fno-omit-frame-pointer' '-march=x86-64' '-mtune=sandybridge' '-msse2')
```

Comment:
According to the Arch RFC, we should do

```suggestion
CFLAGS_COMMON_ARCH=('-fno-omit-frame-pointer' '-march=x86-64' '-mtune=sandybridge' '-msse2' 'mno-omit-leaf-frame-pointer')
```